db/schema.rb

```diff
@@ -17,15 +17,15 @@
   enable_extension "plpgsql"
```

Comment:
I think we won't need these changes, nothing was changed in the database in this PR right?